### PR TITLE
Fix(charts): the qps and brust of reconcile client configurable

### PR DESCRIPTION
**Pull request by yangsoon**
Signed-off-by: yangsoon <songyang.song@alibaba-inc.com>

### Description of your changes

Fixes #

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: charts/vela-core/values.yaml [hunk 1]
```diff
@@ -100,6 +100,12 @@ definitionRevisionLimit: 20
 # concurrentReconciles is the concurrent reconcile number of the controller
 concurrentReconciles: 4
 
+restClient:
```
Comment (wonderflow) [inline charts/vela-core/values.yaml:103]:
```suggestion
kubeClient:
```

File: charts/vela-minimal/values.yaml [hunk 2]
```diff
@@ -106,6 +106,12 @@ dependCheckWait: 30s
 # OAMSpecVer is the oam spec version controller want to setup
 OAMSpecVer: "minimal"
 
+restClient:
```
Comment (wonderflow) [inline charts/vela-minimal/values.yaml:109]:
```suggestion
kubeClient:
```

**Review by wonderflow — APPROVED** [review]
LGTM after nit

File: charts/vela-minimal/values.yaml (hunk 2 above)
Comment (yangsoon) [reply charts/vela-minimal/values.yaml]: thanks, fix it

**Review by leejanee — APPROVED** [review]

**Review by hongchaodeng — APPROVED** [review]

**Pull request merged**